rest-service/manager_rest/models.py

```diff
@@ -277,3 +277,18 @@ def __init__(self, **kwargs):
         self.excluded_wheels = kwargs['excluded_wheels']
         self.supported_py_versions = kwargs['supported_py_versions']
         self.uploaded_at = kwargs['uploaded_at']
+
+
+class Event(SerializableObject):
```

Comment:
event/logs docs may contain variable fields, and do not comply to a single schema
we'll have to consult and see if this solution is sufficient